src/hw9/ArraySizeException.java

```diff
@@ -1,6 +1,6 @@
 package hw9;
 
-public class ArraySizeException extends Throwable{
-    public ArraySizeException ( String s ){
+public class ArraySizeException extends Throwable {
```

Comment:
Extending from `Throwable` is not really correct. Would be enough to extend from `RuntimeException`